src/angular-pdfjs-viewer.js

```diff
@@ -150,7 +150,7 @@
                     });
 
                     if (shouldDigest) $scope.$apply();
-                }, 200, 0, false);
+                }, 500, 0, false);
```

Comment:
Is this to apply after window resize? If so maybe 1500?

Reply:
it also affects the initialization, changed to 1000